polyfill/lib/duration.mjs

```diff
@@ -287,7 +291,8 @@ export class Duration {
         roundingIncrement,
         smallestUnit,
         roundingMode
-      ).duration;
+      ));
+      return ES.UnnormalizeDuration(duration, largestUnit);
```

Comment:
I considered "denormalize", but it has a [particular meaning](https://en.wikipedia.org/wiki/Subnormal_number) in floating point numbers that I thought could be confusing, since normalizing and unnormalizing durations is related to floating point operations.

I agree "unnormalize" isn't ideal so if you have other suggestions than "denormalize", let's rename this operation in a follow-up PR.